prime-router/settings/wa-doh.yml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Again, can't tell if this is correct because I can't tell if this is destined for -local, -test, or -prod.